test/lib/mayaUsd/fileio/utils/testXformStack.py

```diff
@@ -15,33 +15,32 @@
 # limitations under the License.
 #
 
-from pxr import Gf
-from pxr import Usd
-
 import mayaUsd.lib as mayaUsdLib
 
+from pxr import Usd
+
 from maya import cmds
 from maya import standalone
 from maya.api import OpenMaya as OM
 
+import fixturesUtils
+
 from collections import OrderedDict
 import itertools
-import os
-import pprint
 import sys
 import unittest
 
 
-class testUsdMayaXformStack(unittest.TestCase):
+class testXformStack(unittest.TestCase):
+
     @classmethod
     def setUpClass(cls):
-        standalone.initialize('usd')
-        cmds.loadPlugin('pxrUsd')
+        fixturesUtils.setUpClass(__file__)
 
     @classmethod
     def tearDownClass(cls):
         standalone.uninitialize()
-    
```

Comment:
You may want to exclude commit 8e39493fd99d192f130e9d360f30c4b29255c548 or otherwise ignore whitespace when looking through this PR, since it causes GitHub to identify `testXformStack.py` as a "large diff".